tutor/src/screens/course-roster/index.jsx

```diff
@@ -84,15 +89,11 @@ class CourseRoster extends React.Component {
       return (
           <CoursePage
               className="roster"
-              title="Course roster"
               course={course}
+              titleBreadcrumbs={this.renderTitleBreadcrumbs()}
+              titleAppearance="light"
+              controlBackgroundColor='white'
```

Comment:
this is fine, but there's really no reason to add the `renderTitleBreadcrumbs` method that I see. it could be rendered inline as:
```jsx
<CoursePage
   titleBreadcrumbs={<CourseBreadcrumb course={this.course} currentTitle="Course Roster" noBottomMargin />}
/>
```